### DISCUSSION TITLE
how  is Mw calculated

### DISCUSSION THREAD

**Discussion by FURUNZ** [SeisSol/SeisSol#1023] [General]
How is Mw calculated during the software calculation, based on the stress of the fault multiplied by the amount of slip, or based on the inversion of the reseivers point on the fault
![屏幕截图 2024-01-28 113229](https://github.com/SeisSol/SeisSol/assets/92135909/06c24590-6e64-48bf-a97a-0e29e922048b)

**Comment by daisy20170101** [comment]
Hi FURUNZ, here is how the magnitude and energy partition is calculated in SeisSol: https://seissol.readthedocs.io/en/latest/energy-output.html
